classes/requests/class-ledyer-request.php

```diff
@@ -76,7 +76,22 @@ private function token() {
 
 		$client_credentials = ledyer()->credentials->get_credentials_from_session();
 
-		$api_auth_base = $this->is_test() ? 'https://auth.sandbox.ledyer.com/' : 'https://auth.live.ledyer.com/';
+
+		$api_auth_base = 'https://auth.live.ledyer.com/';
+
+		if ( $this->is_test() ) {
+			switch (ledyer()->get_setting( 'development_test_environment' )) {
+				case 'local':
+					$api_auth_base = 'http://host.docker.internal:9001/';
```

Comment:
Like discussed, it would be nice to be able to start locally without having to spin up a BE-service locally